pkg/gatewayserver/gatewayserver.go

```diff
@@ -612,6 +612,9 @@ func requireDisconnect(connected, current *ttnpb.Gateway) bool {
 			return true
 		}
 	}
+	if len(connected.Antennas) > 0 && len(current.Antennas) > 0 && connected.Antennas[0].Gain != current.Antennas[0].Gain {
```

Comment:
Please fix the code quality comment, and consider comparing all of the antennas in the same way `sameAntennaLocations` works.

Reply:
Is it necessary to compare all antennas considering that the antenna gain is used for downlink and we currently support downlinks only via one antenna?

Reply:
Yes, because we shouldn't expand the list things that we need to fix in order to make multiple antenna downlinks work later on, if we want to fix it at some point.